src/isolatable-hybrid.store.ts

```diff
@@ -39,8 +53,25 @@ export class IsolatableHybridConfigurationStore<T> extends HybridConfigurationSt
       (persistentStoreIsExpired && servingStoreIsEmpty);
 
     if (updateServingStore) {
-      this.servingStore.setEntries(entries);
+      this.setServingStoreEntries(entries);
     }
     return updateServingStore;
   }
+
+  /**
+   * Hook to rehydrate serving store entries either from the persistent store or explicitly.
+   * Useful for manually triggering a serving store update when the upstate strategy is not set to always.
+   */
+  private async rehydrate(entries?: Record<string, T>): Promise<void> {
```

Comment:
unfortunately, this is only partially rehydrating, as the config metadata is not being set. It's probably fine though for this use case